Cargo.toml

```diff
@@ -33,27 +33,27 @@ optional = true
 version = "0.3"
 
 [dependencies.digest]
-version = "0.6"
+version = "0.7"
 
 [dependencies.subtle]
 version = "^0.3"
 default-features = false
 
 [dependencies.generic-array]
 # same version that digest depends on
-version = "^0.8"
```

Comment:
The `^` character is redundant in this case, as semver is the default matching used by cargo